aptos-move/aptos-transactional-test-harness/tests/aptos_test_harness/borrow_in_loop.exp

```diff
@@ -1,7 +1,55 @@
 processed 4 tasks
 
 task 1 'publish'. lines 4-26:
-Error: VMError with status STLOC_UNSAFE_TO_DESTROY_ERROR at location Module ModuleId { address: f75daa73fc071f93593335eb9033da804777eb94491650dd3f095ce6f778acb6, name: Identifier("m") } at index 0 for function definition at code offset 11 in function definition 0
+Error: VMError with status STLOC_UNSAFE_TO_DESTROY_ERROR at location Module ModuleId { address: f75daa73fc071f93593335eb9033da804777eb94491650dd3f095ce6f778acb6, name: Identifier("m") } and message Backtrace: Backtrace [
```

Comment:
Maybe you accidentally included a debugging stack trace here?

Reply:
Yep, I forgot that I currently always need to do `(unset RUST_BACKTRACE; UB=1 cargo test)`